src/sdk.ts

```diff
@@ -157,6 +171,70 @@ class APWineSDK {
     return ready
   }
 
+  /**
+   * Switch to signer usage on the sdk instance.
+   * This is necessary if transactions are to be executed.
+   */
+  useSigner() {
```

Comment:
can we do it directly on transaction call instead of separated method?

Reply:
no, we can't because most of the times you create contract instances, and then do transactions on it. If the instance is created without a signer, you will need to recreate it. 

> Hm, what is wrong with the build?

I haven't set up any pipelines so this is some default stuff, which is failing.